### PR TITLE
resolves: args were missing with noNamespaces

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: packages/templates/typescript-resolvers/src/config.ts [hunk 1]
```diff
@@ -6,12 +6,17 @@ import * as afterSchema from './after-schema.handlebars';
...
 
 typescriptConfig.templates['resolver'] = resolver;
 typescriptConfig.templates['schema'] = `${beforeSchema}${typescriptConfig.templates['schema']}${afterSchema}`;
 typescriptConfig.customHelpers.getParentType = getParentType;
 typescriptConfig.customHelpers.getFieldType = getFieldType;
 typescriptConfig.customHelpers.importMappers = importMappers;
+typescriptConfig.customHelpers.getFieldResolver = getFieldResolver;
+typescriptConfig.customHelpers.getFieldResolverName = getFieldResolverName;
+// KAMIL: why? Do we have a default now?
 typescriptConfig.outFile = 'resolvers-types.ts';
```
Comment (kamilkisiela) [inline packages/templates/typescript-resolvers/src/config.ts]: @dotansimha

File: packages/templates/typescript/src/config.ts [hunk 2]
```diff
@@ -36,13 +34,12 @@ export const config: GeneratorConfig = {
...
-    getFieldResolver,
-    getFieldResolverName,
     getOptionals,
     isPrimitiveType,
     getScalarType,
     shouldHavePrefix,
     getEnumValue
   },
+  // KAMIL: why? Do we have a default now?
   outFile: 'types.ts'
```
Comment (kamilkisiela) [inline packages/templates/typescript/src/config.ts]: @dotansimha

Comment (dotansimha) [reply packages/templates/typescript/src/config.ts]: We always had a default :)

Comment (kamilkisiela) [reply packages/templates/typescript/src/config.ts]: really? Didn't notice :))

**Pull request merged**